project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=0.13.13
```

Comment:
As said in https://github.com/RBMHTechnology/eventuate/pull/353#issuecomment-258301692 for 0.13.13 `triggeredBy` is broken due to sbt/sbt#1444, therefore I'd suggest not to upgrade sbt here.